features/networking/operator.feature

```diff
@@ -345,3 +345,25 @@ Feature: Operator related networking scenarios
     And all existing pods die with labels:
       | app=dhcp-daemon |
     """
+
+  # @author anusaxen@redhat.com
+  # @case_id OCP-27333
+  @admin
+  @destructive
+  Scenario: Changing mtu in CNO should not be allowed
+  Given the mtu value "1750" is patched in CNO config according to the networkType
+  And admin uses the "openshift-network-operator" project
+  When I run the :get admin command with:
+    | resource | pods                               |
+    | o        | jsonpath={.items[*].metadata.name} |
+  Then the step should succeed
+  And evaluation of `@result[:response]` is stored in the :network_operator_pod clipboard
```

Comment:
We can use below steps, should be more reliable in case there are multiple pod in the project.
```
And a pod becomes ready with labels:
  | foo=bar |
```
Then call `<%= pod.name %>` in the logs command.

Reply:
yea this is actually a system project and ```a pod becomes ready with labels``` would be called by a testuser so might not work here.